src/sys/select.rs

```diff
@@ -53,6 +53,20 @@ impl FdSet {
             *bits = 0
         }
     }
+
+    /// Finds the largest file descriptor in the set.
+    ///
+    /// Returns `None` if the set is empty.
+    pub fn largest(&self) -> Option<RawFd> {
```

Comment:
Ah, nevermind, it does work. I probably typo'd `#[cfg(test)]` before. But since the other tests for this type are inside `test_select.rs`, I'm going to leave it this way unless you object.